addons/sourcemod/scripting/gokz-replays/recording.sp

```diff
@@ -119,45 +169,83 @@ void OnPlayerRunCmdPost_Recording(int client, int buttons, int tickCount, const
 	recordedRecentData[client].SetArray(tick, tickData);
 }
 
-void GOKZ_OnTimerStart_Recording(int client)
+Action GOKZ_OnTimerStart_Recording(int client)
+{
+	// Hack to fix an exception when starting the timer on the very
+	// first tick after loading the plugin.
+	if (recordedRecentData[client].Length == 0)
+	{
+		return Plugin_Handled;
+	}
+
+	// We are still recording the post-run breather for the previous run,
+	// this means we still did not save the replay, so we would end up
+	// overwriting the data in the buffers.
+	if (postRunRecording[client])
+	{
+		return Plugin_Handled;
```

Comment:
Does returning `Plugin_Handled` prevent the timer from being started? Preventing the timer from being started during the post breather would be very counter-intuitive.

Reply:
> Does returning `Plugin_Handled` prevent the timer from being started? Preventing the timer from being started during the post breather would be very counter-intuitive.

It does, however in practice I don't think anyone will notice. It's only 1 second of not being able to start the timer right after a PB run.
To not do this a lot of code complexity would have to be added.